package.json

```diff
@@ -43,13 +43,13 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "go-ipfs-dep": "0.4.1",
     "ipfs-api": "^4.1.0",
     "multiaddr": "^2.0.0",
     "rimraf": "^2.4.5",
     "run-series": "^1.1.4",
     "shutdown": "^0.2.4",
-    "subcomandante": "^1.0.5"
+    "subcomandante": "^1.0.5",
```

Comment:
Yeah, because despite having a run method (which is poorly named in my opinion) this will only act like a search for the binary and run a test command to check everything is ok (usually `--version` or `version` in our case), not considering any outputs and such. So we still need something to run the command itself and, as we already use `subcomandante`, I assume it's probably the best choice.